app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    _, source_file, destination_path = command.split()
+    if "/" in destination_path:
+        os.makedirs(os.path.dirname(destination_path), exist_ok=True)
```

Comment:
Directories cannot be created if you use certain symbols in names. Does your code handle this?